org.dashbuilder/dashbuilder-widgets/.zanata-cache/etag-cache.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<etag-cache/>
```

Comment:
Do we need these Zanata cache files? If not, could you please remove them from this PR and add the .zanata-cache dir into .gitginore?